docs/docs/tutorial/part-1/index.mdx

```diff
@@ -87,6 +99,12 @@ cd Desktop
 gatsby new
 ```
 
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+**Having trouble with `gatsby new`?** If you had trouble globally installing Gatsby in Part 0, you can also create a new site by running `npm init gatsby` from the command line instead of `gatsby new`.
```

Comment:
```suggestion
**Having trouble with `gatsby new`?** If you had trouble globally installing `gatsby-cli` in Part 0, you can also create a new site by running `npm init gatsby` from the command line instead of `gatsby new`.
```